template/steps/model_evaluator.py

```
@@ -79,8 +79,8 @@
         for message in messages:
             logger.warning(message)
 
-    log_artifact_metadata(
-        metadata={"train_accuracy": float(trn_acc), "test_accuracy": float(tst_acc)},
-        artifact_name="sklearn_classifier",
-    )
+    client = Client()
+    latest_classifier = client.get_artifact_version("sklearn_classifier")
+    log_metadata(metadata=metadata, artifact_version_id=latest_classifier.id)
```

Ruff (0.8.0)

Undefined name `log_metadata`

(F821)